test/python/circuit/test_circuit_qasm.py

```diff
@@ -246,6 +247,22 @@ def test_circuit_qasm_with_composite_circuit_with_many_params_and_qubits(self):
 
         self.assertEqual(original_str, qc.qasm())
 
+    def test_c3sxgate_roundtrips(self):
+        """Test that C3SXGate correctly round trips.  Qiskit gives this gate a different name
+        ('c3sx') to the name in Qiskit's version of qelib1.inc ('c3sqrtx') gate, which can lead to
+        resolution issues."""
+        qc = QuantumCircuit(4)
+        qc.append(C3SXGate(), qc.qubits, [])
+        qasm = qc.qasm()
+        expected = """OPENQASM 2.0;
+include "qelib1.inc";
+qreg q[4];
+c3sqrtx q[0],q[1],q[2],q[3];
+"""
```

Comment:
Haha, my preference is directly the opposite - the suggested form is difficult to both read and write, since you've had to swap the string quotes on one line to avoid a clash, and there's a lot of syntactic noise (extra indents, quotes, and colons).  When I had to edit a bunch of the tests in the OQ3 exporter, it was really inconvenient to have them in the form you're suggesting because editing was very fiddly.  I think a single different offset (on a line that's just boilerplate anyway) is worthwhile trade-off for everything else being easier.